tests/contrib/test_multistore_file.py

```diff
@@ -48,6 +48,24 @@ def filename(self):
         return self.filename_str
 
 
+class Test__dict_to_tuple_key(unittest.TestCase):
+
+    def test_key_conversions(self):
+        d = {'somekey': 'some value', 'another': 'something else',
```

Comment:
For maximum orthodoxy, never repeat your test data values. Consider:
```
first_key = 'somekey'
<...>
third_value = 'foo'
test_dictionary = {
    first_key: first_value,
    second_key: second_value,
    third_key: third_value,
}
```
and then later:
```
expected_tuple = (
    (first_key, first_value),
    (second_key, second_value),
    (third_key, third_value),
)
```
.

Okay, okay, okay: orthodoxy is fussy and sometimes verbose. But at least develop a habit of _feeling bad_ when repeating test data values?